spatial/r2/vector.go

```diff
@@ -40,6 +42,18 @@ func (p Vec) Cross(q Vec) float64 {
 	return p.X*q.Y - p.Y*q.X
 }
 
+// Norm returns the euclidian norm of p:
```

Comment:
We generally don't use a colon in this place. Either a comma, or nothing. and a full stop after the formula on the next line.

Also s/euclidian/Euclidean/g.